docs/package.json

```diff
@@ -7,6 +7,8 @@
   "devDependencies": {
     "@nuxt/content-theme-docs": "^0.7.2",
     "nuxt": "^2.14.7",
+    "sass": "^1.30.0",
```

Comment:
*suggestion*: with my suggestion above about accessing the theme colours, you don't need sass and more generally I would advise not to add in extra dependencies in a pr of this kind

Reply:
I will be careful for the future. 
In fact, with usage of tailwindcss we don't need to use sass and inject extra dependencies 👍